packages/server/src/auth/auth.controller.ts

```diff
@@ -37,11 +48,13 @@ export class RecoveryController {
   constructor(private authService: AuthService) {}
 
   @Post('forgot')
+  @UsePipes(new ForgotPasswordTransformPipe())
   async forgotPassword(@Body() user: ForgotDto): Promise<void> {
     return this.authService.forgotPassword(user.projectId, user.email);
   }
 
   @Post('reset')
+  @UsePipes(new ResetPasswordTransformPipe())
```

Comment:
The resolver will also need the transform added to it

Reply:
What's the resolver?

Reply:
The GraphQL resolver which handles the GraphQL requests (`auth.resolver`). In NestJS you should be able to use the pipes the same way on resolver methods